Sources/Sentry/SentryBreadcrumb.m

```diff
@@ -3,8 +3,48 @@
 #import "NSDictionary+SentrySanitize.h"
 #import "SentryLevelMapper.h"
 
+@interface
+SentryBreadcrumb ()
+@property (atomic, strong) NSDictionary<NSString *, id> *_Nullable unknown;
+@end
+
 @implementation SentryBreadcrumb
 
+- (nullable instancetype)initWithDictionary:(NSDictionary *_Nullable)dictionary
```

Comment:
I wonder about the `nullable` here... of course, `[super init]` could always return `nil` in certain cases, but usually I see `init` methods declared in nonnull regions, so unless this method would explicitly return `nil` for failure to validate the input dictionary, maybe this and the user init should be nonnull instead.

Reply:
We assumed that we might return nil if there is an error in the dererialization, but this is not the case with the current implementation. I will update the API to refelct that. Also, passing a nil dictionary also does not make sense as an API.